templates/zerver/help/add-custom-profile-fields.md

```diff
@@ -40,6 +42,36 @@ There are several different types of fields available.
 * **Person picker**: For selecting other users, like "Manager" or
     "Direct reports".
 
+## Display custom fields in user profile summaries
+
+Organizations may find it useful to display additional fields in a user's
+profile summary, such as pronouns, GitHub username, job title, team, etc.
+
+All field types other than "Long text" or "Person" have a checkbox option
+that controls whether to display a custom field in a user's profile summary.
+
+There's a limit to the number of custom profile fields that can be displayed
+at a time. If the maximum number of fields is already selected, all unselected
+checkboxes will be disabled.
+
+{start_tabs}
+
+{settings_tab|profile-field-settings}
+
+1. Click the pencil (<i class="fa fa-pencil"></i>) icon on the profile field
+   you want to edit.
```

Comment:
I think we want to use **pencil** (with bold formatting) here.

Reply:
We should do a sweep in the help center at some point to make sure we are using bold formatting consistently. Maybe we should use bold formatting for the examples [listed here](https://zulip.readthedocs.io/en/stable/documentation/helpcenter.html#icons ) too.

Reply:
Yeah, I did a sweep for "pencil", but not for other icons, so that would be a good follow-up.

> Maybe we should use bold formatting for the examples [listed here](https://zulip.readthedocs.io/en/stable/documentation/helpcenter.html#icons) too.

Good idea!